src/lwt_support.mli

```diff
@@ -23,12 +23,22 @@ val really_write : Lwt_unix.file_descr -> string -> unit Lwt.t
 (** Read from an Lwt fd and send the chunks to an iteratee *)
 val lwt_fd_enumerator : Lwt_unix.file_descr -> 'a t -> 'a t Lwt.t
 
-(** Read from a names file and send the chunks to an iteratee *)
+(** Read from a named file and send the chunks to an iteratee *)
 val lwt_enumerator : string -> 'a t -> 'a t Lwt.t
 
 exception Host_not_found of string
 
-(** given a host and a port, open a TCP connection and return a connected
-    file descriptor *)
-val open_connection_fd : string -> int -> Lwt_unix.file_descr Lwt.t
+(** Given a file descriptor (Lwt), it executes the function 
+    [callback] passing it the connected file descriptor, ensuring to close
+    the file descriptor when returning. *)
```

Comment:
It might be useful to mention here that any exceptions that happen when closing the file are suppressed.

Reply:
But actually the situation is very complicated here, as ignoring the exception in the cleanup function is required to preserve the original exception. So the doc is fine as it is.